dashboard/app/email_test.go

```diff
@@ -158,7 +158,7 @@ For more options, visit https://groups.google.com/d/optout.
 		c.expectEQ(msg.Headers["In-Reply-To"], []string{"<1234>"})
 		c.expectEQ(msg.Body, fmt.Sprintf(`syzbot has found a reproducer for the following issue on:
 
-HEAD commit:    10101010 a really long title, longer than 80 chars, really..
+HEAD commit:    101010101010 a really long title, longer than 80 chars, really..
```

Comment:
If we increase commit hash here, we need to proportionally shorten title. Otherwise it will wrap in 80-column clients.

Reply:
Hmm.. Didn't think about it. Fixed.